packages/frontend/editor-ui/src/stores/users.store.ts

```diff
@@ -153,11 +144,9 @@ export const useUsersStore = defineStore(STORES.USERS, () => {
 		addUsers([user]);
 		currentUserId.value = user.id;
 
-		const defaultScopes: Scope[] = [];
-		RBACStore.setGlobalScopes(user.globalScopes || defaultScopes);
-		telemetry.identify(rootStore.instanceId, user.id);
-		postHogStore.init(user.featureFlags);
-		npsSurveyStore.setupNpsSurveyOnLogin(user.id, user.settings);
+		loginHooks.value.forEach(async (hook) => {
+			await hook(user);
+		});
```

Comment:
1. See async comment above. 
2. I _think_ a regular for-loop is more performant then `forEach`.
3. A hook that fails will prevent subsequent ones from running? Is that okay?

Reply:
Very good points. I've updated the code.